src/components/ContractName.tsx

```diff
@@ -1,17 +1,15 @@
 import { Suspense, memo } from 'react'
-import { useSnapshot } from 'valtio'
 import Network from 'models/Network'
-import SealCredStore from 'stores/SealCredStore'
 import classNamesToString from 'helpers/classNamesToString'
 import classnames, {
   display,
   flexWrap,
   minWidth,
   wordBreak,
 } from 'classnames/tailwind'
-import getContractName from 'helpers/getContractName'
-import prettifyContractName from 'helpers/prettifyContractName'
-import truncateMiddleIfNeeded from 'helpers/truncateMiddleIfNeeded'
+import getContractName from 'helpers/network/getContractName'
+import prettifyContractName from 'helpers/network/prettifyContractName'
```

Comment:
Smells like we don't need those two separate helpers, maybe we can chain them like this:
```TS
const contractName = getContractName({prettify: true, truncate: truncateOption})
```
?